src/main/cli/cli.c

```diff
@@ -5305,9 +5281,6 @@ static void printConfig(char *cmdline, bool doDiff)
             cliPrintHashLine("start the command batch");
             cliPrintLine("batch start");
 #endif
-
-            cliPrintHashLine("name");
-            printName(dumpMask, &pilotConfig_Copy);
```

Comment:
We could probably still print a `set name = <name>` up here - this is intended to make it simple for users to identify what craft this backup is from.